hack/pr-deploy.yaml

```diff
@@ -199,3 +199,158 @@ objects:
         - name: config
           configMap:
             name: config
+- apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
```

Comment:
This is badly indented - looks like pretty much everything from L203 below should be shifted right.

Reply:
I fixed the indentation issue